foundry.toml

```diff
@@ -21,6 +21,7 @@ fs_permissions = [
 ] # Configures permissions for cheatcodes that touch the filesystem
 extra_output = ["metadata", "abi", "bin"]
 extra_output_files = ["metadata", "abi", "bin"]
+ignored_error_codes = [5574]
```

Comment:
Why? Comment

Reply:
removed in latest push, was trying to convert to 0.8.25 but got a bunch of errors